view/index.js

```diff
@@ -6,14 +6,20 @@ var ViewGenerator = module.exports = function ViewGenerator(args, options, confi
   // By calling `NamedBase` here, we get the argument to the subgenerator call
   // as `this.name`.
   yeoman.generators.NamedBase.apply(this, arguments);
-
-  console.log('Creating a new ' + this.name + ' view.');
+  
+  console.log('Creating a new ' + this.name + ' view (flat: ' + this.flatHierarchyViews + ').');
```

Comment:
`console.log('Creating a new ' + this.name + ' view (' + this.viewHierarchy + ').');`